packages/manager/apps/pci-load-balancer/src/pages/create/steps/network/parts/SubnetNetworks.part.tsx

```diff
@@ -33,9 +32,14 @@ export const SubnetNetworksPart = (): JSX.Element => {
     store.subnet?.id,
   );
 
+  const {
+    data: gateway,
+    isFetching: isGatewayFetching,
+  } = useSmallestGatewayByRegion(store.region?.name);
+
   return (
     <>
-      {isSubnetGatewaysFetching ? (
+      {isSubnetGatewaysFetching || isGatewayFetching ? (
```

Comment:
```suggestion
      if(isSubnetGatewaysFetching || isGatewayFetching) {
         return <OsdsSpinner inline />
     }
     
     return [...]
```

Avoid too many encapsulation